app/src/main/res/layout/dialog_link_preview.xml

```diff
@@ -117,6 +117,23 @@
                     android:layout_gravity="center_horizontal"
                     android:layout_marginTop="4dp" />
 
+                <com.google.android.material.chip.Chip
```

Comment:
What's the reason for making this a `Chip` instead of a `MaterialButton` like elsewhere in the app? We have a number of predefined button styles called `App.Button.*` for you to use, one of which is the exact style that's needed here, without any other changes.